data.txt

```diff
@@ -1,8 +1,66 @@
+ "amount": "1234.0"  "description": "Hello World"  "paymentMethod": "Cash"  "time": "01/01/1900"  "category": "Food" 
+ "amount": "1234.0"  "description": "Hello World"  "paymentMethod": "Cash"  "time": "01/01/1900"  "category": "Food" 
+ "amount": "1234.0"  "description": "Hello World"  "paymentMethod": "Cash"  "time": "01/01/1900"  "category": "Food" 
+ "amount": "1234.33"  "description": "欠�?"  "paymentMethod": "Cash"  "time": "01/01/1900"  "category": "Food" 
+ "amount": "1234.66"  "description": "欠�?"  "paymentMethod": "Cash"  "time": "01/01/1900"  "category": "Food" 
+ "amount": "0.33"  "description": "欠�?"  "paymentMethod": "Cash"  "time": "01/01/1900"  "category": "Food" 
+ "amount": "1234.0"  "description": "欠�?"  "paymentMethod": "Cash"  "time": "01/01/1900"  "category": "Food" 
+ "amount": "1234.68"  "description": "test"  "paymentMethod": "Cash"  "time": "01/01/1900"  "category": "Food" 
+ "amount": "1234.32"  "description": "test"  "paymentMethod": "Cash"  "time": "01/01/1900"  "category": "Food" 
+ "amount": "1234.0"  "description": "Hello World"  "paymentMethod": "Cash"  "time": "01/01/1900"  "category": "Food" 
+ "amount": "1234.0"  "description": "\\\"); System.out.println(\"helo\");"  "paymentMethod": "Cash"  "time": "01/01/1900"  "category": "Food" 
+ "amount": "1234.0"  "description": "\\;"  "paymentMethod": "Cash"  "time": "01/01/1900"  "category": "Food" 
+ "amount": "1234.0"  "description": "\"asd\"); System.out.println(\"helo\");"  "paymentMethod": "Cash"  "time": "01/01/1900"  "category": "Food" 
+ "amount": "1234.0"  "description": "欠�?"  "paymentMethod": "Cash"  "time": "01/01/1900"  "category": "Food" 
 # END EXPENDITURES
 # BEGIN CREDIT CARDS
- "totalExpenditure": "0.0"  "monthlyCardLimit": "500.0"  "nameOfCard": "OCBC"  "cashback": "1.5" 
+ "totalExpenditure": "0.0"  "monthlyCardLimit": "1.0E8"  "nameOfCard": "[CREDIT_CARD_NAME]"  "cashback": "99.99" 
+ "totalExpenditure": "0.0"  "monthlyCardLimit": "1.0E8"  "nameOfCard": "[CREDIT_CARD_NAME]"  "cashback": "99.99" 
+ "totalExpenditure": "0.0"  "monthlyCardLimit": "1.0E8"  "nameOfCard": "[CREDIT_CARD_NAME]"  "cashback": "99.99" 
+ "totalExpenditure": "0.0"  "monthlyCardLimit": "1.0E8"  "nameOfCard": "[CREDIT_CARD_NAME]"  "cashback": "0.1" 
+ "totalExpenditure": "0.0"  "monthlyCardLimit": "1.0E8"  "nameOfCard": "[CREDIT_CARD_NAME]"  "cashback": "1.0E-9" 
+ "totalExpenditure": "0.0"  "monthlyCardLimit": "1.0E8"  "nameOfCard": "[CREDIT_CARD_NAME]"  "cashback": "1.0E-74" 
+ "totalExpenditure": "0.0"  "monthlyCardLimit": "1.0E25"  "nameOfCard": "[CREDIT_CARD_NAME]"  "cashback": "1.0E-74" 
+ "totalExpenditure": "0.0"  "monthlyCardLimit": "1.0E8"  "nameOfCard": "|||"  "cashback": "10.0" 
+ "totalExpenditure": "0.0"  "monthlyCardLimit": "1.0E8"  "nameOfCard": "&&&&&&&&"  "cashback": "10.0" 
 # END CREDIT CARDS
 # BEGIN INCOME SOURCES
+ "amount": "123123"  "category": "Salary" 
```

Comment:
Perhaps can remove most of the test cases in the .txt file before uploading so that there is a clean slate of .txt file when the others pull from the updated code.